xtevent/xtevent.ado

```diff
@@ -138,18 +139,23 @@ program define xtevent, eclass
 	if "`repeatedcs'"!=""{
 		di as txt _n "Option {bf:repeatedcs} was specified. Using {bf:`panelvar'} as the panel variable and {bf:`timevar'} as the time variable."
 	}
-
-	if ("`cohort'" != "" & "`control_cohort'" == "") | ("`cohort'" == "" & "`control_cohort'" != "")  {
-		di as err _n "options {bf:cohort} and {bf:control_cohort} must be specified simultaneously"
-		exit 199
-	}
-	if "`cohort'"!="" & "`control_cohort'"!=""  {
+	
+	if "`cohort'"!="" {

```

Comment:
@jorpppp 
when I combine IV specification and Sun-and-Abraham estimation, for instance:
```
xtevent y eta , panelvar(i) timevar(t) policyvar(z) window(10) impute(stag) proxy(x) proxyiv(2) sunabraham 
xtevent y eta , panelvar(i) timevar(t) policyvar(z) window(10) impute(stag) proxy(x) proxyiv(2) cohort(create) 
```
 `sunabraham` and `cohort(create)` are passed as options to `_eventiv` but not making difference to the IV estimation because we have not implemented Sun and Abraham for the IV setting. Maybe we can add an error message to inform the user that she/he cannot combine `proxy` and `cohort`, for instance.

Reply:
@Constantino-Carreto-Romero Agreed. I'll implement. Note that enabling IV estimation with SA estimation is pending in #125

Reply:
Solved in https://github.com/JMSLab/xtevent/pull/185/commits/181fc2ef9c437a55dbe2c9c60f79b45289fc873a